docs/src/pages/components/alert/demo.tsx

```diff
@@ -1,35 +1,153 @@
-import React from 'react';
-import { Alert, Flex } from '@aws-amplify/ui-react';
+import { useState } from 'react';
+import Highlight, { defaultProps } from 'prism-react-renderer';
+import {
+  Alert,
+  Tabs,
+  TabItem,
+  Card,
+  Flex,
+  defaultTheme,
+  AmplifyProvider,
+  createTheme,
+  Theme,
+  View,
+  Button,
+} from '@aws-amplify/ui-react';
 
-import { AlertPropControls } from '@/components/AlertPropControls';
-import { useAlertProps } from '@/components/useAlertProps';
-import { Example } from '@/components/Example';
+import { AlertPropControls } from './AlertPropControls';
+import { useAlertProps } from './useAlertProps';
+import { AlertThemeControls, useAlertThemeProps } from './AlertThemeControls';
+
+const alertTheme = defaultTheme.tokens.components.alert;
+
+const propsToCode = (props) => `import { Alert } from '@aws-amplify/ui-react';
```

Comment:
I don't know if there is a better way to do this. We could use sandpack and have this as an editable example, but would be less performatn.